test/reflection_test.cpp

```diff
@@ -605,4 +605,35 @@ TEST(Reflection, MockFailingAddressOfOperator)
 
 }  // namespace fixed_containers
 
+namespace fixed_containers::reflection::customize
+{
+struct MyCustomStruct
+{
+    int a{};
+    int b{};
+    int c{};
+};
+
+template <>
+struct ReflectionHelper<MyCustomStruct>
+{
+    static constexpr auto FIELD_NAMES = make_fixed_vector<std::string_view>({
+        "a",
+        "b",
+        "c",
+    });
+
+    template <typename T2, typename Func>
+    static constexpr void apply(T2&& instance, Func&& func)
+    {
+        func(FIELD_NAMES.at(0), instance.a);
+        func(FIELD_NAMES.at(1), instance.b);
+        func(FIELD_NAMES.at(2), instance.c);
```

Comment:
you should make the custom `apply` and `FIELD_NAMES` impl here do something different than the default one so the test actually proves that we're using it instead of just classic reflection

Reply:
Added a 4th field that is intentionally ignored and comment to that effect.